test/langtools/jdk/javadoc/doclet/testGenericTypeLink/pkg1/A.java

```diff
@@ -60,6 +60,11 @@ public void someMethod(List<? extends Number> list, int i) {}
      */
     public void otherMethod(Map<String, ?> list, double d) {}
 
+    /**
+     * @see A<String, java.lang.RuntimeException>.Inner
+     * @see A<A<String, java.lang.RuntimeException>.Inner, A.SomeException>
```

Comment:
Is this a new feature that we can link with type arguments? Interesting, as https://docs.oracle.com/en/java/javase/16/docs/specs/javadoc/doc-comment-spec.html#see has no mention on usage of type arguments in see links.